persistence/persistence-api/src/main/java/org/cbioportal/persistence/StructuralVariantRepository.java

```diff
@@ -4,14 +4,16 @@
 
 import org.cbioportal.model.GeneFilterQuery;
 import org.cbioportal.model.StructuralVariant;
+import org.cbioportal.model.StructuralVariantIdentifier;
 import org.springframework.cache.annotation.Cacheable;
 
 public interface StructuralVariantRepository {
 
     @Cacheable(cacheResolver = "generalRepositoryCacheResolver", condition = "@cacheEnabledConfig.getEnabled()")
     List<StructuralVariant> fetchStructuralVariants(List<String> molecularProfileIds,
                                                     List<String> sampleIds,
-                                                    List<Integer> entrezGeneIds);
+                                                    List<Integer> entrezGeneIds, 
+                                                    List<StructuralVariantIdentifier> structuralVariants);
```

Comment:
```suggestion
                                                    List<StructuralVariantIdentifier> structuralVariantIds);
```

Reply:
Renamed in [4c93d75](https://github.com/cBioPortal/cbioportal/commit/4c93d75879595db2e4add0ff8452a5647b540756)